### PR TITLE
TWIRF6 - CFP finalised

### PR DESCRIPTION
**Pull request by dns2utf8**
This is my first draft.

### CODE REVIEW

File: _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "This week in RustFest 6: CFP finalisation"
```
Comment (vasia) [inline _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md]: I'm not sure "CFP finalization" is very accurate. How about "Talk selection" or "Schedule finalization?". Finalizing the _call_ for proposals is not the same as selecting talks, no?

File: _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+authors:
+  - dns2utf8
+---
+
+This week we finalised the CFP. It was amazing to see many quality proposals from a variety of areas, disciplines, and backgrounds. This weekend we spent hours trying to put together a topic spanning conference lineup building on topics we haven't covered at Rustfest, as well as talks featuring the evolution of the story we continue to follow in topics we have visited in previous Rustfests. We're excited to send out mails to new speakers as well as returning speakers.
```
Comment (vasia) [inline _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md]: The second sentence "This weekend ... previous RustFests" is quite long and confusing. How about the following:

This weekend we spent hours reviewing proposals and putting together the conference lineup. We did our best to cover both topics we yet haven't seen at Rustfest, as well as talks featuring the evolution of topics we have previously visited.

File: _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+![XKCD Joke involving email backlog. Stick person 1: "Do you have any New Years Resolutions", Stick person 2: "Gonna finally finish dealing with those emails from 2008", Slugline: "As my Email backlog approaches 10 years, I' m starting to have doubts about my approach."](https://imgs.xkcd.com/comics/emails.png)
+
+We will be sending out emails to the accepted speakers over the next few days. If you submitted something please check your `SPAM` and `JUNK` folders if you haven't heard anything by Tuesday evening.
+
+In case you submitted a proposal
```
Comment (vasia) [inline _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md]: Is there something missing here?

**Review by vasia — COMMENTED** [review]
Left a few minor comments. Thank you ^^

File: _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md (hunk 3 above)
Comment (Hoverbear) [reply _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md]: Definitely. :-P

File: _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md (hunk 1 above)
Comment (Hoverbear) [reply _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md]: Fixed

File: _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md (hunk 2 above)
Comment (Hoverbear) [reply _posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md]: Fixed

**Comment by dns2utf8** [comment]
Thank you for your work 👍

**Pull request merged**